pkg/cloudprovider/providers/aws/aws.go

```diff
@@ -969,7 +968,9 @@ func init() {
 			})
 
 		aws := newAWSSDKProvider(creds)
-		return newAWSCloud(*cfg, aws)
+		cloud, err := newAWSCloud(*cfg, aws)
+		registerMetrics(cloud)
+		return cloud, err
 	})
```

Comment:
One problem with this approach is - this will cause emission of these metrics (whenever possible) from nodes too. won't that cause any problems?

Reply:
Yeah that’s bad, let me try to shuffle code around to avoid this